### PR TITLE
Budget app

### PR DESCRIPTION
**Pull request by getssh**
Interactions
Splash screen

A simple page with the name of your app (yes, you need to choose one), and links to the sign up and log in pages.
Sign up and log in pages

The user should be able to register in the app with full name, email and password (all mandatory).
The user can log into the app using email and password.
If the user is not logged in, they can't access pages that require the user to be logged in (all the pages described below).
Home page (categories page)

When the user logs in, they are presented with the categories page.
For each category, the user can see their name, icon and the total amount of all the transactions that belongs to that category.
When the user clicks (or taps) on a category item, the application navigates to the transactions page for that category.
There is a button "add a new category" at the bottom that brings the user to the page to create a new category.
Transactions page

For a given category, the list of transactions is presented, ordered by the most recent.
At the top of the page the user could see the total amount for the category (sum of all of the amounts of the transactions in that category).
There is a button "add a new transaction" at the bottom that brings the user to the page to create a new transaction.
When the user clicks on the "Back" button (<), the user navigates to the home page.
"Add a new category" page

The user fills out a form to create a new category, indicating their name and icon (both mandatory).
The user clicks (or taps) the "Save" button to create the new category, and is taken to the home page on success.
When the user clicks on the "Back" button (<), the user navigates to the home page.
"Add a new transaction" page

The user fills out a form to create a new transaction with:
name (mandatory)
amount (mandatory)
categories (mandatory at least one)
The user click (or taps) the "Save" button to create the new transaction, and is taken to the transactions page for that category.
When the user clicks on the "Back" button (<), the user navigates to the transactions page for that category.
Testing requirements
Create unit and integration tests for all the most important components of your RoR application.
Technical requirements
You should use Postgres as your database.
You should use devise for authentication.
You should validate all user input to make sure that anyone with bad intentions cannot compromise your app.
You can use a view template engine of your choice (.erb, .slim, .haml).
The project should be deployed and accessible online.
Your database schema should reflect the following structure:

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,169 @@
...
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project you can give it a star!
+
+<p align="right"><a href="#readme-top">Back to top</a></p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
```
Comment (rloterh) [inline README.md:170]: - [x] It's a nice and smart thing to give appropriate credit to **[Gregoire Vella on Behance](https://www.behance.net/gregoirevella)**, the author of [the original design](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)), as required by the [Creative Commons License](https://creativecommons.org/licenses/). Kindly ensure to add this under the **Acknowledgements** section of your ReadMe.

File: README.md [hunk 2]
```diff
@@ -0,0 +1,169 @@
...
+  - [📝 License ](#-license-)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 Rails Budget App <a name="about-project"></a>
+
+**Budget app** An appliation help to track users expenses. The app has authunication for security reasons.
+
+<a href="https://www.loom.com/share/a1e6cae6108346f9a3e74d6c53418c0f?sid=3c9ddcfe-e7ab-4f19-9488-f871d2b4db1a">Presentation vedio about the project</a>
+
```
Comment (rloterh) [inline README.md:34]: - [x] Your app is apparently not deployed, as live demo link cannot be found in the ReadMe. Kindly ensure to/that:
  * deploys the application using Render, Heroku or similar for Rails app.
  * a link to the deployed project is present in the ReadMe.

**Review by rloterh — CHANGES_REQUESTED** [review]
Hi Getayawkal, 

You did extremely great on this project! 👏 👏 
There are however a few issues that you still need to work on to go to the next project, but you are almost there! 
🏁 🚦 🏃 🌴 🏫 🌳 🏘️

## Highlights 👍  
- Properly added required models, controllers, and corresponding views
- Devise correctly implemented for authentication!
- Schema follow given ERD!
- App is running as intended!

## Required Changes ♻️
Your app is working fine, good job! 👏 However, it will be a good idea if we can add a few tweaks to the design aspect so as to improve the look and feel. Please take note that it's one of the project's objectives that our design closely mimics the given [template design](https://www.behance.net/gallery/19759151/Snapscan-iOs-design-and-branding?tracking_source=). Let's improve upon the design aspect by introducing the following tweaks:

- [x] Let's remove the black background from the back button in the header section, similar to the original design.
- [x] Let's ensure that layout on the splash screen is similar to the original design.
- [x] Let's ensure that the green button is always sticky to the bottom for category and transactions pages, and has an inner white border  similar to the original design.
- [x] Let's ensure that the typography in your design matches the one used in the original design.


_Kindly check the other comments under the review._

Cheers and Happy coding!👏 💻 🍷

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 2 above)
Comment (rotimiazeez) [reply README.md:34]: Kindly implement the change as requested here 👍🏽 You should have a link to the deployed version of your app in your README file 💯

File: README.md (hunk 1 above)
Comment (rotimiazeez) [reply README.md:170]: **Please note that this is part of the project requirement** Kindly make sure appropriate credit to [Gregoire Vella on Behance](https://www.behance.net/gregoirevella), the author of [the original design](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)), as required by the [Creative Commons License](https://creativecommons.org/licenses/).

**Review by rotimiazeez — CHANGES_REQUESTED** [review]
Hi @getssh  😉, 

Great work on making the changes 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


## Suggested changes
_Check the comments under the review._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me(@rotimiazeez) in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid, unless it is requested otherwise._**

**Comment by rloterh** [comment]
@getssh Can you try using render for hosting?

**Review by anagudelogu — APPROVED** [review]
# STATUS: APPROVED! 🍾 🎆 🥇 

Hi @getssh, 
I'm Andres.

You did a great job working on this milestone, your project is complete! 🚀 
There is nothing else to say other than... it's time to merge it 🤖  
Congratulations! 🎉 



## Highlights
✅ Great use of Git flow. 💯 
✅ Your project is documented professionally. 📜 
✅ Great UI design. 💻 
✅ Descriptive PR title, summary and commit messages. 💬 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**